### DISCUSSION TITLE
Question: Why not using compiled parsers

### DISCUSSION THREAD

**Discussion by dragove** [nvim-treesitter/nvim-treesitter#5782] [Q&A] [answered]
I don't know why it is designed to build parsers locally, is compiled parsers not working?
In Emacs, I can simply download compiled parsers from [tree-sitter-langs](https://github.com/emacs-tree-sitter/tree-sitter-langs) and place this binaries to `~/.emacs.d/tree-sitter` folder after renaming. It's much easier to use, less time on compilation and download(since the binary is much smaller than source codes)
The question is why not binary.

**Comment by clason** [comment, marked as answer]
Because we (Neovim) support a much wider range of platforms (and parsers) than Emacs; it's just not feasible for us to provide every combination. (Note that tree-sitter-langs doesn't provide binaries for every commit, either, while nvim-treesitter has a rolling release model.)

Once WASM parser support has stabilized (which was recently added to tree-sitter master), we'll investigate how to distribute those.
